setup.php

```diff
@@ -55,37 +45,84 @@
 require_once dirname(__FILE__) . '/library/classes/Installer.class.php';
 
 $state = isset($_POST["state"]) ? ($_POST["state"]) : '';
-
+$installer1 = new Installer($_REQUEST);
```

Comment:
Why are we using 2 different Installer instances?
(see line 149)

Reply:
Should be able to drop the 1 on line 149 and call this one $installer

Reply:
done. I had created a new object because the original one had been declared after a some conditionals  and I had not wanted to mess with the logic

Reply:
Done.